inc/utils.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <string>
+#include <ctime>
+#include "StatusCode.hpp"
+
+namespace utils {
+
+const int timeStringBuffer = 80;
```

Comment:
Macros are more c style way of doing things. 
They don't give you static type checking, since the Preprocessor just takes the macro value and inserts it everywhere it can find the macro. 
By using const int, the compiler can do a type check beforehand and warn if needed. This might not apply here, but is more of a coding style. For example in Core Guidelines ES.31: [Don’t use macros for constants or “functions”](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions)